simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/support/HexUtils.java

```diff
@@ -0,0 +1,29 @@
+package com.evbox.everon.ocpp.simulator.station.support;
+
+public class HexUtils {
+
+    public static boolean isNotHex(String value) {
```

Comment:
what about regex? [\dA-F]{40} :-)

Reply:
slower :)

Reply:
where are you checking the  [\dA-F]**{40}** bolded part of his Regex in your code?

Reply:
`if (invalidLength(attributeValue) || isNotHex(attributeValue.toString())) {`